UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -81,7 +63,7 @@
               "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
               "valueExpression": {
                 "language": "text/cql",
-                "expression": "\"UrologicalSuppliesPrepopulation\".AnswerYes"
+                "expression": "\"BasicClinicalInfoPrepopulation\".AnswerYes"
```

Comment:
Now I see where that code in the CQL was being used. This doesn't make sense. If you want to prepopulate with a static value, then `item.initial` should be used instead of CQL.

Reply:
fixed